src/Provider/URLBuilder.js

```diff
@@ -94,7 +94,7 @@ export default {
      * @return {string} the formed url
```

Comment:
I would prefer naming the new parameter as `bboxPrecision` or `bboxDigits` to conform to the naming of the parameter of function [`toFixed`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed).

Reply:
Ok I rename to `bboxDigits`